runtime/src/accounts_db.rs

```diff
@@ -100,6 +100,9 @@ pub type SnapshotStorages = Vec<SnapshotStorage>;
 // Each slot has a set of storage entries.
 pub(crate) type SlotStores = HashMap<usize, Arc<AccountStorageEntry>>;
 
+type PurgedAccountSlots = HashMap<Pubkey, HashSet<Slot>>;
```

Comment:
similar to https://github.com/solana-labs/solana/pull/12606/files#r498668833, how about `AccountSlots`/`purged_account_slots`

Reply:
Fixed in latest commit! https://github.com/solana-labs/solana/pull/12606/commits/69092648864a17e4a21d5699a51b5c2b6b83c16e